Makefile

```diff
@@ -1,5 +1,6 @@
 CC = g++
-CPPFLAGS = -g -Wall -O3 -I. -Iglog-src -Wno-sign-compare
+CPPFLAGS = -g -Wall -O3 -I. -Iglog-src -Wno-sign-compare -fpermissive
```

Comment:
What's the error you get without `-fpermissive`?

Reply:
It looks like the errors occur in glog-src/demangle.cc - "error: ISO C++ forbids comparison between pointer and integer [-fpermissive]". I'm merging in the upstream changes.